src/BookmarksPanel.svelte

```diff
@@ -32,11 +36,33 @@
         anchor.click();
         URL.revokeObjectURL(url);
     }
+
+    function updateColorSelection() {
+        // Toggle whether or not to change/export node colors
+        let cb = document.getElementById("color-toggle");
+
+        if (cb.checked === true) {
+            document.getElementById("color-selection").disabled = false;
```

Comment:
Note that this only activates the first element on the page, so rows ≥2 are still disabled. See this example:

<img width="810" alt="Screen Shot 2022-09-29 at 3 08 34 PM" src="https://user-images.githubusercontent.com/320904/193121132-a31d0773-dfdf-4052-8ca7-d89585ff97da.png">

Furthermore, try to avoid using `getElementById(...)` and other native DOM APIs. Use Svelte binding instead – see my comment below on the color input.